<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteWidget</class>
 <widget class="QWidget" name="PaletteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pinkBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="orangeBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yellowBtn">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>10</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="purpleBtn">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lightBlueBtn">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="redBtn">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="darkGreenBtn">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="blueBtn">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lightGreenBtn">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="goldenBtn">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
